app/models/foreman_tasks/lock.rb

```diff
@@ -120,7 +120,7 @@ def link?(resource, uuid)
 
       # Records the information about the user that triggered the task
       def owner!(user, uuid)
-        build_owner(user, uuid).save!
+        # TODO: Log some deprecation warning?
```

Comment:
Can we drop this method all together with other unused code or still the todo is valid?